.github/workflows/ci.yml

```diff
@@ -164,7 +164,7 @@ jobs:
               (cd $dir && terraform init -backend-config="path=$dir")
               (cd $dir && terraform fmt -check)
               (cd $dir && terraform validate)
-              (cd $dir && terraform plan -input=false)
+              (cd $dir && terraform plan -input=false -var='project_id=something' -var='bucket_name=something')
```

Comment:
This is an intended change? 😄

Reply:
Yes. Our CI requires some kind of value to be passed in here for it to work as anticipated. it doesn't actually do anything since we're only running `terraform plan` there, so the values don't matter.